packages/ndla-ui/src/global/components/component.glossary-word.scss

```diff
@@ -59,11 +60,23 @@
   padding: $spacing;
   animation-name: fadeInTop;
   animation-duration: .4s;
+  opacity: 0;
+}
+.c-glossary-word__popup--visible {
+  opacity: 1;
 }
 .c-glossary-word__title {
   margin-top: 0;
   border-bottom: 2px solid $brand-color;
   text-transform: capitalize;
+  display: block;
+  font-weight: bold;
```

Comment:
Foreslår å legge til en klasse `.u-heading3` i  _elements.headings.scss og bruk `{...classes('title', undefined, 'u-heading3')}` . Si ifra om det er uklart.

Reply:
Hvis jeg forstår det rett brukes `undefined` for å unngå at det blir lagt på "noe ekstra" etter "title" i klassenavnet, og parameter nr. 3 er et ekstra klassenavn som ikke har noe med det første å gjøre?

Reply:
Jepp, display må være. Angående undefined så stemmer det også.